app/static/_css/style.css

```diff
@@ -254,4 +254,24 @@ body>.navbar + * {
 .fixed-section,
 .settings-content {
     padding-top: 30px;
+}
+
+/* READING LIST */
+#readinglist .img-container {
+    position: relative;
+    height: 250px;
+    overflow: hidden;
+    width: 100%;
+    margin-bottom: 5px;
+    border-radius: .2rem;
+}
+
+#readinglist h4 {
+    padding: 0;
+    margin: 0;
+    font-weight: bold;
+}
+
+#readinglist h2 {
+    margin-top: 100px;
 }
```

Comment:
Please always make sure that there is an empty newline at the end of each file.
Why? See https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

- [ ] Add newline